github-orgs/kubeflow/org.yaml

```diff
@@ -1217,3 +1217,12 @@ orgs:
             privacy: closed
             repos:
               xgboost-operator: write
+          kubeflow-outreach-committee:
+            description: Kubeflow Outreach Committee
+            members:
+            - andreyvelich
```

Comment:
You can remove myself and add @varodrig and @chasecadet to this GitHub team.
cc @kubeflow/kubeflow-steering-committee